### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by meteora2002**
finished the json file in simple language

this is my github contribution .feel free to give constructive feedbacks

### CODE REVIEW

**Comment by tuladhar** [comment]
Congrats @meteora2002 on your first contribution. I suggest you to make README a bit more readable using correct formatting, currently, it does look rightly formatted.

**Comment by meteora2002** [comment]
done

**Comment by meteora2002** [comment]
@tuladhar please accept my pull requests

File: README.md [hunk 1]
```diff
@@ -1028,13 +1028,60 @@ To Start using the sqlite3 library:
...
+
+
 ## `Exploring JSON Module` exploring-json-module
 
 JSON is text, written with JavaScript object notation. JSON is a syntax for storing and exchanging data.
-
+It is commonly used for transmitting data in web applications (e.g., sending some data from the server to the client, so it can be displayed on a web page
+and vice versa
 - `json.loads()`
+-json. load() takes a file object and returns the json object. A JSON object contains data in the form of key/value pair. The keys are strings and the values are the JSON types
```
Comment (tuladhar) [inline README.md]: Please remove the duplicate mention of json.load(), for e.g:

```suggestion
- `json.loads()` takes a file object and returns the json object. A JSON object contains data in the form of key/value pair. The keys are strings and the values are the JSON types
```

and remove duplicate json.dumps() in below lines as well.

File: README.md [hunk 2]
```diff
@@ -1028,13 +1028,60 @@ To Start using the sqlite3 library:
...
 - `json.dumps()`
+-json.dumps() function converts a Python object into a json string
+.It is the exact opposite of json.loads.
+
+## THIS IS THE ENCODING DECODING LIST
```
Comment (tuladhar) [inline README.md]: Let's use table formatting for encoding and decoding list.

**Review by tuladhar — CHANGES_REQUESTED** [review]

**Comment by tuladhar** [comment]
Thanks @meteora2002 I've requested for some changes, please check.

**Comment by meteora2002** [comment]
@tuladhar please accept my pull requests

**Comment by meteora2002** [comment]
@tuladhar  please check my pull request

File: README.md [hunk 3]
```diff
@@ -1028,13 +1028,54 @@ To Start using the sqlite3 library:
...
+It is commonly used for transmitting data in web applications (e.g., sending some data from the server to the client, so it can be displayed on a web page
+and vice versa
 - `json.loads()`
+- take a file object and returns the json object. A JSON object contains data in the form of key/value pair. The keys are strings and the values are the JSON types
 - `json.dumps()`
+-json.dumps() function converts a Python object into a json string
+.It is the exact opposite of json.loads.
+
+## THIS IS THE ENCODING DECODING LIST
+|  JSON|  Python |
```
Comment (tuladhar) [inline README.md]: Table is still not showing up correctly.

File: README.md [hunk 4]
```diff
@@ -1028,13 +1028,54 @@ To Start using the sqlite3 library:
...
-
+It is commonly used for transmitting data in web applications (e.g., sending some data from the server to the client, so it can be displayed on a web page
+and vice versa
 - `json.loads()`
+- take a file object and returns the json object. A JSON object contains data in the form of key/value pair. The keys are strings and the values are the JSON types
 - `json.dumps()`
+-json.dumps() function converts a Python object into a json string
+.It is the exact opposite of json.loads.
+
+## THIS IS THE ENCODING DECODING LIST
```
Comment (tuladhar) [inline README.md]: Let's use normal text for it, and remove the heading formatting.

File: README.md [hunk 5]
```diff
@@ -1028,13 +1028,54 @@ To Start using the sqlite3 library:
...
+
+
+ - `  Encoding is from python to JSON(final type)`
+>>> json.JSONEncoder().encode({"foo": ["bar", "baz"]})
+'{"foo": ["bar", "baz"]}'
+
+
+- `Decoding is from JSON to python(final type)`
+
+## other commands
```
Comment (tuladhar) [inline README.md]: Let's rename it to "Encoding functions"

File: README.md [hunk 6]
```diff
@@ -1028,13 +1028,54 @@ To Start using the sqlite3 library:
...
+- `iterencode(o)`
+-Encode the given object, o, and yield each string representation as available. For example:
+>>>for chunk in json.JSONEncoder().iterencode(bigobject):
+    mysocket.write(chunk)]
+` -sort-keys` -Sort the output of dictionaries alphabetically by key.
+`-h, --help¶`- help box 
+`infile`-to check your Json file for syntax
+`outfile`-Write the output of the infile to the given outfile
 
+## note
```
Comment (tuladhar) [inline README.md]: Let's capitalize it.

**Review by tuladhar — CHANGES_REQUESTED** [review]

**Comment by meteora2002** [comment]
done

**Comment by meteora2002** [comment]
@tuladhar please check my pull request

File: README.md [hunk 7]
```diff
@@ -1028,13 +1028,55 @@ To Start using the sqlite3 library:
...
+and vice versa
 - `json.loads()`
+- take a file object and returns the json object. A JSON object contains data in the form of key/value pair. The keys are strings and the values are the JSON types
 - `json.dumps()`
+-json.dumps() function converts a Python object into a json string
+.It is the exact opposite of json.loads.
+
+THIS IS THE ENCODING DECODING LIST
+
+|  JSON |Python |
```
Comment (tuladhar) [inline README.md]: Table is still not showing. You can learn about it here:
- https://www.makeuseof.com/tag/create-markdown-table/

**Review by tuladhar — CHANGES_REQUESTED** [review]

**Comment by tuladhar** [comment]
Also, @meteora2002 let's remove the "coming soon" label from the contents table for JSON and add link.

Screen Shot 2020-10-13 at 12 05 57

File: README.md [hunk 8]
```diff
@@ -16,7 +16,7 @@
   6. [Argparse Module](#exploring-argparse-command-line-option-and-argument-parsing)
   6. [SQLite Module](#exploring-standard-modules-embedded-relational-database-module) ![Generic badge](https://img.shields.io/badge/Coming%20-Soon-blue)
   7. [XmlTree Module](#exploring-xmltree-module) ![Generic badge](https://img.shields.io/badge/Coming%20-Soon-blue)
-  8. [JSON Module](#exploring-json-module) ![Generic badge](https://img.shields.io/badge/Coming%20-Soon-blue)
```
Comment (tuladhar) [inline README.md:19]: Please fix the link and point to:
- #exploring-json-module-exploring-json-module

**Review by tuladhar — CHANGES_REQUESTED** [review]

**Review by tuladhar — APPROVED** [review]
Thanks @meteora2002 for your contributions. Looks good to merge!

**Pull request merged**